pages/view-hazard/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../../assets/css/variables.css">
+  <link rel="stylesheet" href="../../assets/css/global.css">
+  <!-- <script type="module" src="../js/home.js" defer></script> -->
+  <link rel="manifest" href="../../manifest.json">
+  <link rel="apple-touch-icon" href="../../assets/css/img/icons/icon-96x96.png">
+  <meta name="apple-mobile-web-app-status-bar" content="#000000">
+  <meta name="theme-color" content="#000000">
+  <!-- leaflet JS for maps -->
+  <link rel="stylesheet" href="../../modules/leaflet/leaflet.css">
+  <script src="../../modules/leaflet/leaflet.js"></script>
+  <script type="module" src="./script.js" defer></script>
```

Comment:
I think these two
```
  <script type="module" src="./script.js" defer></script>
  <link rel="stylesheet" href="./style.css">
```

should be at last. taking priority over the other styles/js